File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        left = 0

        for i in range(len(nums)):
            if nums[i] != val:
                nums[left], nums[i] = nums[i], nums[left]
                left += 1
        
        return left

        

File: 27-remove-element/remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int left = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                int tmp_left = nums[left];
                nums[left] = nums[i];
                nums[i] = tmp_left;
                left++;
            }
        }

        return left;
    }
}